PayPal/src/main/java/com/braintreepayments/api/paypal/PayPalCheckoutRequest.kt

```diff
@@ -63,6 +63,8 @@ import org.json.JSONObject
  * at this URL.
  *
  * @property contactInformation Contact information of the recipient for the order
+ *
+ * @property contactPreference Preference for the contact information section within the payment flow
```

Comment:
Should we include something like "if not set, the contact information section will be disabled in the payment flow"? That's the behavior we see if we don't pass this param at all right?

Reply:
Yep, on the API side the enum will default to `NO_CONTACT_INFO` if no value is passed in. I've updated the doc string.